core/src/main/java/org/elasticsearch/index/query/InnerHitBuilder.java

```diff
@@ -47,7 +47,7 @@
 
 import static org.elasticsearch.common.xcontent.XContentParser.Token.END_OBJECT;
 
-public final class InnerHitBuilder extends ToXContentToBytes implements Writeable {
+public final class InnerHitBuilder implements Writeable, ToXContentObject {
```

Comment:
this applies to all the classes that don't extend from ToXCOntentToBytes anymore and don't declare their own toString .